src/app/PCM/templates/navigation/tpl-north.js

```diff
@@ -46,34 +45,33 @@ const TemplateNorth = ({ bg, color, mappedPcm, logoSrc }) => {
                   <FontAwesomeIcon size="lg" icon={faSearch} cursor="pointer" color="#234371" />
                 </HStack>
               </Box> */}
-            <Box mx={5} alignItems="center" m="auto">
-              {apiConfig && (
-                <Image
-                  onClick={() =>
-                    onSendMessage({
-                      code: dashboardViewQuestion,
-                      parentCode: dashboardViewQuestion,
-                    })
-                  }
-                  src={logoSrc}
-                  htmlWidth={LOGO_WIDTH}
-                />
-              )}
-            </Box>
-            <Spacer />
-            <HStack spacing={8} marginRight="5">
-              <AskMenu
-                questionCode={PRI_LOC2}
-                icon={<Button leftIcon={<FontAwesomeIcon icon={faPlus} />}>{`Add`}</Button>}
-                hideLabel={true}
+          <Box mx={5} alignItems="center" m="auto">
+            {apiConfig && (
+              <Image
+                onClick={() =>
+                  onSendMessage({
+                    code: dashboardViewQuestion,
+                    parentCode: dashboardViewQuestion,
+                  })
+                }
+                src={logoSrc}
+                htmlWidth={LOGO_WIDTH}
               />
-              <Drafts code={PRI_LOC3} />
-              <Avatar code={PRI_LOC4} />
-            </HStack>
-          </Flex>
-        </nav>
-      </header>
-    </>
+            )}
+          </Box>
+          <Spacer />
+          <HStack spacing={8} marginRight="5">
+            <AskMenu
+              questionCode={PRI_LOC2}
+              icon={<Button leftIcon={<FontAwesomeIcon icon={faPlus} />}>{`Add`}</Button>}
```

Comment:
The information about which icon to use comes from the PCM, we cannot have the icon specified ourselves.